### ISSUE TITLE
Windows V8: cannot require('leveldown')

### ISSUE THREAD

**Issue by ktrzeciaknubisa** [jxcore/jxcore#569]
The installation goes fine with JXcore 0.3.0.7 (for older versions you should add `--build-from-source` to the command below):

```bash
$ jx install leveldown
```

However `require('leveldown')` fails with the following error:

```bash
Error: The specified module could not be found.
c:\jxcore\jxcore-issues\jxcore\x564\node_modules\leveldown\build\release\leveldown.node
    at Object.process.dlopen [as .node] (node.js:666:19)
    at Module.load (module.js:416:30)
    at Function.Module._load (module.js:380:8)
    at Module.require (module.js:451:15)
    at require (module.js:469:13)
    at bindings (C:\jxcore\jxcore-issues\jxcore\x564\node_modules\leveldown\node_modules\bindings\bindings.js:76:44)
    at Object.<anonymous> (C:\jxcore\jxcore-issues\jxcore\x564\node_modules\leveldown\leveldown.js:4:46)
    at Module._compile (module.js:595:24)
    at Object.Module._extensions..js (module.js:625:8)
    at Module.load (module.js:416:30)
```
The mentioned path for *leveldown.node* file **is correct**.

Not sure, if this is any tip, but by default *leveldown/build/config.gypi* uses: 
`"nodedir": "C:\\Users\\nubisa_krzs\\.node-gyp\\0.10.40"`.

Now, I'm doing the following:

```bash
$ cd node_modules/leveldown
$ jx C:/Users/nubisa_krzs/.jx/npm/node_modules/node-gyp/bin/node-gyp.js rebuild
```

And after that `require('leveldown')` **works fine** (does not crash and exits with code 0).

Is this because the new *config.gypi* file uses different path: 
`"nodedir": "C:\\Users\\nubisa_krzs\\.node-gyp\\jxb304"`?

**Comment by obastemur** [comment]
thats the reason. we might overwrite nodedir?

**Comment by ktrzeciaknubisa** [comment]
Overwriting `nodedir` didn't bring any good. Only bunch of errors. However I've found another way. 

The problem originated from the fact, that [prebuild](https://github.com/mafintosh/prebuild) modules was using [node-gyp-install](https://github.com/mafintosh/node-gyp-install) to fetch node header files, saving them into e.g. `~/.node_gyp/0.10.40` (depending on `process.version`) and then using for compilation with its own node-gyp. 

All this is triggered by `install` step defined in leveldown's *package.json*:

```json
  "scripts": {
    "install": "prebuild --download"
  }
```

Now, I figured, that replacing `prebuild --download` into `node-gyp rebuild` not only skips downloading prebuilt binaries (`--build-from-source` seems no longer needed in this case), but also runs the compilation against our own `node-gyp` and `~/.node_gyp/jxbXXX`.

It's already working for me, on Windows too. 
Just few more tests, and we're ready very soon with it.
